src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public double getS() {
+        return 3.14 * Math.pow(this.radius, 2);
+    }
+
+    @Override
+    public void drawFigure() {
+        System.out.println("Коло { радіус = " + this.radius + ", площа = "
+                            + getS() + ", колір - " + getColor() + " }");

```

Comment:
The `drawFigure()` method contains non-English text. According to the checklist, you should use English for messages to ensure the code is universal and consistent.